docs/source/models/cause.rst

```diff
@@ -337,14 +336,38 @@ Mortality Impacts
 Cause-Specific Mortality
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
+The cause-specific mortality rate is the mortality rate from a specified cause for a population. 
+The numerator is the number of deaths attributed to a specific cause. The denominator remains the 
+size of the population. In GBD, cause-specific estimates by country, year, age, sex.
+
+  For example, 112 measles deaths in country A during calendar year 2016. 1010,623 estimated 2016 mid-year 
+  population. 11.08 measles deaths per 100,000 population (person-years at risk) during calendar year 
+  2016 in country A. 
+
 Excess Mortality
 ^^^^^^^^^^^^^^^^
 
+The excess mortality is obtained by subtracting the expected mortality from the observed mortality.
+The excess mortality can be interpreted as the mortality directly or indirectly related to the disease of 
+interest, if one excludes all other causes of death. It is the extra number of deaths observed, beyond that 
+expected, per person-years.   
+
+  For instance, an excess mortality rate of 40 deaths per 100 person-years means that there are 40 additional 
+  deaths per 100 person-years in excess of those expected. These deaths are supposed to be related to the disease 
+  of interest.
+
 Morbidity Impacts
 +++++++++++++++++
 
 Disability Weights
 ^^^^^^^^^^^^^^^^^^
 
+A disability weight is a weight factor that reflects the severity of the disease on a scale from 0 (perfect health) to 1 
+(equivalent to death). Years Lost due to Disability (YLD) are calculated by multiplying the incident cases by duration and 
```

Comment:
this logic is the "incidence DALY", and we now prefer the "prevalence DALY", where the disability weight is multiplied by the person-years lived with condition.

Reply:
changed to prevalence